LEAF_Request_Portal/templates/reports/LEAF_table_input_report.tpl

```diff
@@ -70,126 +67,74 @@ function populateIndicators(categoryID)
+                    text = text.length <= 55 ? text : text.slice(0, 55) + '...';
+                    $('select#dataField').append($('<option />').val(data[i].indicatorID).text(text));
                     indicatorFormats[data[i].indicatorID] = JSON.parse(format.substring(5));
                 }
             }
-            if(!anyTables)
-            {
+            if(!anyTables) {
                 $('div#dataFieldContainer').hide();
                 alert('No grid input found in this form.');
-            }
-            else{
+            } else{
                 $('div#dataFieldContainer').show();
             }
         }).fail(function (jqXHR, error, errorThrown) {
             console.log(jqXHR);
             console.log(error);
             console.log(errorThrown);
         });
-    }
-    else
-    {
+
+    } else {
         $('div#dataFieldContainer').hide();
     }
 }
 
 //build the header array
 //these will be the headers for the current version of the grid
-function buildHeaderArray(columnNames)
-{
+function buildHeaderArray(columnNames) {
     headerArray = [];
-    if(typeof columnNames !== 'undefined')
-    {
-        for(let i = 0; i < columnNames.length; i++)
-        {
+    if(typeof columnNames !== 'undefined') {
+        for(let i = 0; i < columnNames.length; i++) {
             headerArray.push(columnNames[i].name);
         }
     }
 }
 
-//get all submitted records using this form
-function getDataForExport(categoryID, indicatorID)
-{
-    processedRecords = 0;
-    $.ajax({
-        type: 'GET',
-        url: './api/form/_'+categoryID+'/records'
-    }).done(function(data) {
-        bodyObj = {};
-        for(let i = 0; i < data.length; i++)
-        {
-            if(data[i].submitted != 0)
-            {
-                bodyObj[data[i].recordID] = [];
-            }
-        }
-        recordIDs = Object.keys(bodyObj);
-        for(let i = 0; i < recordIDs.length; i++)
-        {
-            addRecordData(recordIDs[i], indicatorID)
-        }
-    }).fail(function (jqXHR, error, errorThrown) {
-        console.log(jqXHR);
-        console.log(error);
-        console.log(errorThrown);
-    });
-}
-
 //get data for the selected indicator associated with the given recordID
-function addRecordData(recordID, indicatorID)
-{
-    $.ajax({
-        type: 'GET',
-        url: './api/formEditor/indicator/'+indicatorID,
-        data: {
-            recordID: recordID
-        }
-    }).done(function(data) {
-        let dataRows = data[indicatorID]['value']['cells'];
-        if (dataRows !== null && dataRows !== undefined) {
-            for (let i = 0; i < dataRows.length; i++) {
-                bodyObj[recordID].push(dataRows[i]);
-            }
-            updateProgress();
-        } else {
-            //if the data for this request isn't in table format, just mark as processed
-            updateProgress();
+function addRecordData(recordID, indicatorID, gridInput = {}) {
```

Comment:
Do we want some defaults here?  Additionally, is indicatorID even being used?  If it's here for legacy sake, we may want to indicate that.